build.sbt

```diff
@@ -28,3 +28,11 @@ libraryDependencies ++= {
 Revolver.settings
 
 javaOptions in Revolver.reStart += "-Dconfig.file=application.conf"
+
+// Copy over various properties
```

Comment:
Exactly, this helps with sbt revolver running in a separate VM. It effectively copies over the properties in .sbtopts.